app/src/main/java/com/google/android/samples/socialite/ui/videoedit/VideoEditScreenViewModel.kt

```diff
@@ -150,7 +149,7 @@ class VideoEditScreenViewModel @Inject constructor(
                 .build()
 
         val editedVideoFileName = "Socialite-edited-recording-" +
-            SimpleDateFormat(CameraViewModel.FILENAME_FORMAT, Locale.US)
+            SimpleDateFormat("yyyy-MM-dd-HH-mm-ss-SSS", Locale.US)
```

Comment:
Is there any reason to remove FILENAME_FORMAT from CameraViewModel companion object and set format here?

Reply:
If you look at `CameraModule`, you can see that file names, paths, and types are created separately to be injected externally. In the future, When we refactor VideoEditScreen, we plan to change it to the same injection method and use separate constants to declare the format internally. 

Do you have any good suggestions? 🤔

Reply:
You can do that simply making it as domain layer's usecase or just utils.

Reply:
Personally, I believe that it is appropriate to create UseCases only when handling the business logic of the program (such as chat, camera, video, etc.). On the other hand, providing only DateFormat as a Util seems like a good approach. 
I'll refactor and apply this after the current PR work is completed. Thank you for your suggestions :)